docs/docs/addons-animation.md

```diff
@@ -210,55 +214,48 @@ You can disable animating `enter` or `leave` animations if you want. For example
 
-## Low-level API: `ReactTransitionGroup`
-
-`ReactTransitionGroup` is the basis for animations. It is accessible from `require('react-addons-transition-group')`. When children are declaratively added or removed from it (as in the example above) special lifecycle hooks are called on them.
+* * *
 
-### `componentWillAppear(callback)`
+## Low-level API: ReactTransitionGroup
 
-This is called at the same time as `componentDidMount()` for components that are initially mounted in a `TransitionGroup`. It will block other animations from occurring until `callback` is called. It is only called on the initial render of a `TransitionGroup`.
-
-### `componentDidAppear()`
+**Importing**
 
-This is called after the `callback` function that was passed to `componentWillAppear` is called.
-
-### `componentWillEnter(callback)`
-
-This is called at the same time as `componentDidMount()` for components added to an existing `TransitionGroup`. It will block other animations from occurring until `callback` is called. It will not be called on the initial render of a `TransitionGroup`.
-
-### `componentDidEnter()`
-
-This is called after the `callback` function that was passed to `componentWillEnter` is called.
-
-### `componentWillLeave(callback)`
-
-This is called when the child has been removed from the `ReactTransitionGroup`. Though the child has been removed, `ReactTransitionGroup` will keep it in the DOM until `callback` is called.
+```javascript
+import ReactTransitionGroup from 'react-addons-transition-group' // ES6
+var ReactTransitionGroup = require('react-addons-transition-group') // ES5 with npm
+var ReactTransitionGroup = React.addons.TransitionGroup; // ES5 with react-with-addons.js
+```
 
-### `componentDidLeave()`
+`ReactTransitionGroup` is the basis for animations. When children are declaratively added or removed from it (as in the [example above](#getting-started)), special lifecycle hooks are called on them.
 
-This is called when the `willLeave` `callback` is called (at the same time as `componentWillUnmount`).
+ - [`componentWillAppear()`](#componentwillappear)
+ - [`componentDidAppear()`](#componentdidappear)
+ - [`componentWillEnter()`](#componentwillenter)
+ - [`componentDidEnter()`](#componentdidenter)
+ - [`componentWillLeave()`](#componentwillleave)
+ - [`componentDidLeave()`](#componentdidleave)
 
-### Rendering a Different Component
+#### Rendering a Different Component
 
-By default `ReactTransitionGroup` renders as a `span`. You can change this behavior by providing a `component` prop. For example, here's how you would render a `<ul>`:
+`ReactTransitionGroup` renders as a `span` by default. You can change this behavior by providing a `component` prop. For example, here's how you would render a `<ul>`:
 
 ```javascript{1}
 <ReactTransitionGroup component="ul">
-  ...
+  // ...
```

Comment:
Minor nit, but `// ...` won' work inside a JSX tag. It will just be displayed as text.
I think the original text used `...` just to make it explicit.
Alternatively you could use a real comment although it might look weird: `{/* ... */}`.